sklearn/model_selection/tests/test_split.py

```diff
@@ -1770,17 +1770,16 @@ def test_nested_cv():
     groups = rng.randint(0, 5, 15)
 
     cvs = [
-        LeaveOneGroupOut(),
-        LeaveOneOut(),
-        GroupKFold(n_splits=3),
-        StratifiedKFold(),
-        StratifiedGroupKFold(n_splits=3),
-        StratifiedShuffleSplit(n_splits=3, random_state=0),
+        StratifiedKFold(n_splits=2),
+        StratifiedShuffleSplit(n_splits=2, random_state=0),
```

Comment:
I think we should still keep 2 group-wise CV strategies and 1 group-agnostic CV strategy to test nesting of groups which is the main purpose of this PR:

```suggestion
        LeaveOneGroupOut(),
        StratifiedKFold(n_splits=2),
        GroupKFold(n_splits=3),
```

Reply:
you can press the "Commit suggestion" button if you accept this suggestion. You can then pull the new commit to your local repo with `git pull`.

Reply:
Ok, got it. I hadn't done the comparison. I hope the updated PR description will suffice.